Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Profiles/EyeTrackingDemoSpeechCommandsProfile.asset

```diff
@@ -130,7 +130,7 @@ MonoBehaviour:
       axisConstraint: 0
   - localizationKey: 
     keyword: Come to me
-    keyCode: 13
+    keyCode: 0
```

Comment:
Do we not want the keyCode anymore?

Reply:
Or is `0` some other value?

Reply:
I was testing why the selection didn't work and noticed that keyCode: 13 (Enter) was assigned to both "select" as well as "Come to me". For "come to me", it's not important though, so I removed it. 
![image](https://user-images.githubusercontent.com/32180789/59062336-6f76c680-885a-11e9-86ca-1f0009a3a40c.png)